src/components/Card/CardList.test.tsx

```diff
@@ -11,27 +11,30 @@ jest.mock("react-router-dom", () => ({
 }));
 
 describe("CardList component", () => {
-  // const mockCards = [
-  //   { index: 0, title: "Card 1", description: "This is card 1", src: "src1" },
-  //   { index: 1, title: "Card 2", description: "This is card 2", src: "src2" },
-  //   { index: 2, title: "Card 3", description: "This is card 3", src: "src3" },
-  // ];
-
   it("renders a list of all cards if no category is supplied", () => {
     const params = {};
     jest.spyOn(ReactRouterDOM, "useParams").mockReturnValue(params);
 
-    const { getByRole } = render(<CardList cardItemData={mockCards}/>);
+    const { getByRole } = render(<CardList cardItemData={mockCards} />);
```

Comment:
If you're using `prettier` this should be handled by the `.eslintrc.json` config.